src/SFA-DAS-HMRC-API-Stub/Data/Repositories/Cosmos/FractionCalDateCosmosRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Azure.Documents.Client;

```

Comment:
The fraction calc and fractions should be in the same repository as they logically sit together